backend/app.py

```diff
@@ -1,72 +1,138 @@
-# """
- 
-
-# def get_completion():
-#     chat_completion = openai.ChatCompletion.create(model="gpt-3.5-turbo", messages=[{"role": "user", "content": generic_prompt}])
-#     print(chat_completion.choices[0].message.content)
-
-
-# if __name__ == "__main__":
-#     get_completion()
-
-# PLANTUML code
-# from plantuml import PlantUML
-
-# server = PlantUML(url='http://www.plantuml.com/plantuml/img/',
-#                           basic_auth={},
-#                           form_auth={}, http_opts={}, request_opts={})
-
-# server.processes_file('test.txt', 'test.png')
-
-# DOCUMENT
-
-# from docx import Document
-# from docx.shared import Inches
-# document = Document()
-
-# document.add_heading('Document Title', 0)
-
-# p = document.add_paragraph('A plain paragraph having some ')
-# p.add_run('bold').bold = True
-# p.add_run(' and some ')
-# p.add_run('italic.').italic = True
-
-# document.add_heading('Heading, level 1', level=1)
-# document.add_paragraph('Intense quote', style='Intense Quote')
-
-# document.add_paragraph(
-#     'first item in unordered list', style='List Bullet'
-# )
-# document.add_paragraph(
-#     'first item in ordered list', style='List Number'
-# )
-
-# document.add_picture('test.png', width=Inches(1.25))
-
-# records = (
-#     (3, '101', 'Spam'),
-#     (7, '422', 'Eggs'),
-#     (4, '631', 'Spam, spam, eggs, and spam')
-# )
-
-# table = document.add_table(rows=1, cols=3)
-# hdr_cells = table.rows[0].cells
-# hdr_cells[0].text = 'Qty'
-# hdr_cells[1].text = 'Id'
-# hdr_cells[2].text = 'Desc'
-# for qty, id, desc in records:
-#     row_cells = table.add_row().cells
-#     row_cells[0].text = str(qty)
-#     row_cells[1].text = id
-#     row_cells[2].text = desc
-
-# document.add_page_break()
-
-# document.save('demo.docx')
\ No newline at end of file
+from flask import Flask, request, jsonify
+from plantuml import PlantUML
+from docx import Document
+from docx.shared import Inches
+import openai
+import json
+import random
+import string
+import os
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient
+
+
+openai.api_key = ''
+model_name = 'gpt-3.5-turbo'
+generic_prompt = """
+>>>I have an ITaskHandler interface that implements handler function which is responsible for generating fixed output if the request matches certain requirements. ITaskHandler is implemented by classes AdditionTask, MultiplyTask, DivideTask. Client calls ITaskHandler through a chain of responsibility design pattern. <<<
+
+======
+Intsuction prompt
+Read >>>CONTENT<<< and generate JSON in the below format
+{
+"DesignDoc": {
+  "Intent":,
+  "Pros",
```

Comment:
add solution keyword here, because of this output was not showing solution. Or refer to aashnas new prompt